test/user.js

```diff
@@ -323,11 +380,15 @@ describe('user', () => {
     });
   });
   describe('setup', () => {
+    const email = 'bob@example.com';
```

Comment:
I'm fine with leaving `const` for test values provided that they're scoped to the nearest-applicable `describe`.